bin/describe.ml

```diff
@@ -160,7 +276,14 @@ module Crawl = struct
           >>| List.filter_map ~f:Fun.id)
       >>| List.concat
     in
-    Memo.Build.return (Dyn.List (exes @ libs))
+    Memo.Build.return (exes @ libs)
+
+  (** Builds a [Dyn.t] value that describes the workspace corresponding to the
+      provided dune setup and context *)
+  let workspace_as_dyn (setup : Dune_rules.Main.build_system)
```

Comment:
I would inline this one-liner in its only use below.

Reply:
Done.